tests/test_unit_tests.py

```diff
@@ -54,6 +54,17 @@ def test_stream_read_and_parse(self):
             title = manifest_store["manifests"][manifest_store["active_manifest"]]["title"]
             self.assertEqual(title, "C.jpg")
 
+    def test_stream_read_string_stream(self):
+        reader = Reader("image/jpeg", self.testPath)
```

Comment:
As an aside: The tests use `Reader` without releasing the memory resources (without calling close). In tests it's tolerable, but we may want to use something like this in all Reader tests (with statement), to clean up resources:

```
with open(self.testPath, "rb") as file:
    reader = Reader("badFormat", file)
```

Would you mind making that change on the added tests test_stream_read_string_stream and test_stream_read_string_stream_and_parse?